server/utils/db/connection_pool.py

```
@@ -85,17 +91,25 @@ def get_connection(self):
             logger.error(f"Error with connection: {e}")
             raise
         finally:
             if connection:
+                try:
+                    connection.rollback()
+                    with connection.cursor() as cur:
+                        cur.execute(
+                            "RESET myapp.current_user_id; RESET myapp.current_org_id;"
+                        )
+                    connection.commit()
+                except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)